packages/plugin-image-array-keyboard-response/docs/jspsych-image-array-keyboard-response.md

```diff
@@ -0,0 +1,44 @@
+| stimulus                    | array of string  | *undefined*        | The path of the image files to be displayed.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| stimulus_rect               | array of integer | null               | Set the positionX, positionY, witdh, height of the images in pixels. If left null for width and height (no value specified), then the image will display at its natural height.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| maintain_aspect_ratio       | boolean          | true               | If setting *only* the width or *only* the height and this parameter is true, then the other dimension will be scaled to maintain the image's aspect ratio.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| keyboard_choices            | array of strings | `"NO_KEYS"` | This array contains the key(s) that the subject is allowed to press in order to respond to the stimulus. Keys should be specified as characters (e.g., `'a'`, `'q'`, `' '`, `'Enter'`, `'ArrowDown'`) - see [this page](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values) and [this page (event.key column)](https://www.freecodecamp.org/news/javascript-keycode-list-keypress-event-key-codes/) for more examples. Any key presses that are not listed in the array will be ignored. Specifying `"ALL_KEYS"` means that all keys will be accepted as valid responses. The default value of `"NO_KEYS"` will mean that no responses are allowed. |
```

Comment:
using `choices` instead of `keyboard_choices` would be more consistent with other plugins